lib/utils/export/index.ts

```diff
@@ -4,10 +4,12 @@ import { get } from 'lodash';
 import exportNotes from './export-notes';
 import exportToZip from './to-zip';
 
+import * as T from '../../types';
+
 const filename = 'notes.zip';
 
-const exportZipArchive = () => {
-  return exportNotes()
+const exportZipArchive = (notes: T.Note[]) => {
```

Comment:
adding `notes` here is great because I plan on adding this as an option when logging out if there are unconfirmed changes in the account:

```
There are changes which may not be synchronized with the server.

 - Lose changes and logout?
 - Export affected notes and logout?
 - Don't logout yet?
```